lib/configuration.js

```diff
@@ -167,6 +167,15 @@ var Configuration = function(givenConfig, logger) {
    */
   this._version = version;
   /**
+   * Boolean flag indicating whether or not the configuration instance was able
+   * to find a set of usable credentials to attempt authorization against the
+   * Stackdriver API.
+   * @memberof Configuration
+   * @private
+   * @type {Boolean}
+   */
+  this._lacksValidCredentials = false;
```

Comment:
Not sure how that would work -- it's _checkAuthConfiguration that is responsible for setting the value of _lacksValidCredentials to true if it cannot find a valid set of credentials. The pr has been updated to default to true though